src/models.rs

```diff
@@ -1,4 +1,4 @@
-use super::schema::casbin_rules;
+use super::schema::casbin_rule;
 
 #[derive(Queryable, Identifiable)]
 pub(crate) struct CasbinRule {
```

Comment:
`failed to resolve: use of undeclared crate or module `casbin_rules``
I don't know how to solve this problem.